android/src/main/java/com/inkapplications/android/view.kt

```diff
@@ -0,0 +1,29 @@
+package com.inkapplications.android
```

Comment:
```suggestion
/*
The following code is based on Jake Wharton's Butterknife library.
Sources received from: https://github.com/JakeWharton/butterknife
Copyright Jake Wharton and licensed under:
Apache License Version 2.0, January 2004
http://www.apache.org/licenses/

Modifications include:
 - Update code to run as an extension rather than a class.
*/
package com.inkapplications.android
```

👮‍♀️ 👮‍♀️ 👮‍♀️ 
Credit where credit is due.